tests/unit/test_pandas_numpy.py

```diff
@@ -0,0 +1,132 @@
+#  python -m unittest discover -s tests/unit
+
+import logging
+import sys
+import os
+import unittest
+
+import kensu.pandas as pd
+from kensu.client import ApiClient
+import kensu
+from kensu.utils.kensu_provider import KensuProvider
+from kensu.utils.dsl import mapping_strategies
+from kensu.utils.dsl.mapping_strategies import Strategy
+from kensu.itertools import kensu_list as list
+from kensu import itertools
+
+log_format = '%(asctime)s %(levelname)s %(filename)s:%(lineno)d %(message)s'
+logging.basicConfig(stream=sys.stdout, level=logging.INFO, format=log_format)
+
+def ndarray_to_csv(nd_arr, fn):
+    return pd.DataFrame(nd_arr).to_csv(fn)
+
+class TestPandasNumpy(unittest.TestCase):
+    offline_file = 'pandas_numpy.jsonl'
+    try:
+        os.remove(offline_file)
+    except OSError:
+        pass
+
+    ksu = KensuProvider().initKensu(
+        #api_url="http://somewhere",
+        #auth_token=token,
+        init_context=True,
+        report_in_mem=False,
+        report_to_file=True,
+        offline_file_name=offline_file,
+        logical_naming='File',
+        mapping=True
+    )
+
+
+    def setUp(self):
+        self.ac = ApiClient()
+        self.dataset = 'tests/unit/fixtures/Macroeconomical.csv'
+
+    def read_df(self, index_col=None, header='infer'):
+        return pd.read_csv(self.dataset, sep=';', header=header, index_col=index_col)
+
+    def assert_msg_exists(self, msg, msg2=None):
+        with open(self.offline_file, "r") as f:
+            assert bool([True for l in f.readlines() if msg in l and (msg2 is None or msg2 in l)])
+
+    def assert_ndarray(self, v):
+        assert str(type(v)) == "<class 'kensu.numpy.ndarray'>"
+
+    def test_df_indexed_value(self):
+        out_fname1 = 'test_df_column.indexer.values'
+        out_fname = 'test_df_indexed_list_of_ndarray'
+        dummy_ts_df = pd.read_csv('https://raw.githubusercontent.com/numenta/NAB/master/data/realTweets/Twitter_volume_AMZN.csv', index_col=0) # header=0,
+        # <class 'kensu.pandas.data_frame.DataFrame'>
+
+        l1 = []
+        # here `value` is a column name!
+        value_series = dummy_ts_df.value[:"2015-04-05 00:00:00"]  # <class 'kensu.pandas.data_frame.Series'>
+        # probably not supporting passing list(Series) to constructor of ndarray?
+        v = value_series.values  # ndarray
+        self.assert_ndarray(v)
+        ndarray_to_csv(v, out_fname1)
+        self.assert_msg_exists('Lineage to kensu-py/test_df_column.indexer.values from realTweets/Twitter_volume_AMZN.csv',
+                               '"columnDataDependencies": {"0": ["value"]}}')
+
```

Comment:
example of integration test "pattern":

- kensu-py collector writes to `offline ingestion jsonl file` as usual with `report_to_file=True`
- test helper in `assert_msg_exists` read offline ingestion jsonl file
- we check for certain patterns in the jsonl file (here just if certain line or string exists - seems enough for now, in spark-collector it's fancier - parse and transform jsonl)